### PR TITLE
feat: move `forest-mainnet` and `forest-calibnet` droplets from `/terraform` to `/tf-managed`

### PR DESCRIPTION
**Pull request by aatifsyed**
# Change Summary
- See title

# Reference issue
- #373 

# Follow-up work
- #413
- #412

### CODE REVIEW

File: .github/workflows/deploy-forest-calibnet.yml [hunk 1]
```diff
@@ -1,40 +1,47 @@
...
-concurrency: ci-${{ github.ref }}
-
 on:
   pull_request:
     branches:
       - main
     paths:
-      - 'terraform/forest-calibnet/**'
-      - 'terraform/modules/filecoin_node/**'
+      - 'tf-managed/modules/forest-calibnet/**'
```
Comment (LesnyRumcajs) [inline .github/workflows/deploy-forest-calibnet.yml:6]: Is that the name of the path to the module?

File: .github/workflows/deploy-forest-calibnet.yml [hunk 2]
```diff
@@ -1,40 +1,47 @@
...
+    # This needs to be declared explicitly so that the job is actually
+    # run when moved out of draft.
+    types: [opened, synchronize, reopened, ready_for_review]
   push:
     branches:
       - main
     paths:
-      - 'terraform/forest-calibnet/**'
-      - 'terraform/modules/filecoin_node/**'
+      - 'tf-managed/modules/forest-calibnet/**'
```
Comment (LesnyRumcajs) [inline .github/workflows/deploy-forest-calibnet.yml:15]: Is that the name of the path to the module?

File: .github/workflows/deploy-forest-calibnet.yml [hunk 3]
```diff
@@ -1,40 +1,47 @@
...
 
 jobs:
   deploy-forest-calibnet:
-    name: Deploy
+    env:
+      TF_VAR_slack_enable: true
+      TF_VAR_slack_destination_id: ${{ secrets.SLACK_DESTINATION_ID }}
+      TF_VAR_slack_channel_id: ${{ secrets.SLACK_CHANNEL_ID }}
+      TF_VAR_pagerduty_enable: true
+      TF_VAR_pagerduty_destination_id: ${{ secrets.PAGERDUTY_DESTINATION_ID }}
```
Comment (LesnyRumcajs) [inline .github/workflows/deploy-forest-calibnet.yml:26]: Are those variables needed?

File: .github/workflows/deploy-forest-calibnet.yml [hunk 4]
```diff
@@ -1,40 +1,47 @@
...
         with:
           do_token: ${{ secrets.DO_TOKEN }}
           aws_access_key_id:  ${{ secrets.AWS_ACCESS_KEY_ID }}
           aws_secret_access_key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-          ssh_private_key: ${{ secrets.SSH_PRIVATE_KEY }}
-          working_directory: terraform/forest-calibnet
-          environment: Calibnet
+          r2_access_key: ${{ secrets.R2_ACCESS_KEY }}
+          r2_secret_key: ${{ secrets.R2_SECRET_KEY }}
+          slack_token: ${{ secrets.SLACK_TOKEN }}
```
Comment (LesnyRumcajs) [inline .github/workflows/deploy-forest-calibnet.yml:42]: Are those keys used? R2 is for snapshot downloading, Slack token is for Slack notifications.

File: .github/workflows/deploy-forest-mainnet.yml
Comment (LesnyRumcajs) [inline .github/workflows/deploy-forest-mainnet.yml:1]: same comments as in calibnet workflow

File: tf-managed/live/environments/dev/applications/forest-calibnet/terragrunt.hcl [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+include "root" {
+  path = find_in_parent_folders()
+}
+
+# Load the actual Terraform module
+terraform {
+  source = format("%s/../modules/forest-in-docker-in-droplet", get_parent_terragrunt_dir())
+}
+
+inputs = {
```
Comment (LesnyRumcajs) [inline tf-managed/live/environments/dev/applications/forest-calibnet/terragrunt.hcl:12]: Let's have it consistent with other services, and have the droplet size here explicitly. Same for the name of the service.

File: tf-managed/modules/forest-in-docker-in-droplet/main.tf
Comment (LesnyRumcajs) [inline tf-managed/modules/forest-in-docker-in-droplet/main.tf:1]: The name of the module is a bit verbose - how about naming it without exposing internal details, like you would with a regular function? For example, `forest-vps`.

**Review by LesnyRumcajs — COMMENTED** [review]
Looking good.

File: tf-managed/modules/forest-droplet/main.tf [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+
+// This needs to be its own resource so that terraform recreates the droplet when it changes
+resource "local_sensitive_file" "bootstrap_script" {
+  filename = "bootstrap.bash"
+  content = templatefile(
+    "${path.module}/bootstrap.bash.tftpl",
+    {
+      NEW_USER             = "user" // TODO(aatifsyed): refactor bootstrap script to not require this
+      CHAIN                = var.chain
+      NR_LICENSE_KEY       = "" // var.NR_LICENSE_KEY // TODO(aatifsyed): plug this functionality
```
Comment (LesnyRumcajs) [inline tf-managed/modules/forest-droplet/main.tf:20]: Let's have those comments tracked by issues; lingering TODOs are seldom addressed.

**Review by LesnyRumcajs — APPROVED** [review]
Please use the PR template for documentation purposes, and let's create a follow-up issue for https://github.com/ChainSafe/forest-iac/issues/373, for monitoring purposes.

File: tf-managed/modules/forest-droplet/main.tf [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+  }
+}
+
+// This needs to be its own resource so that terraform recreates the droplet when it changes
+resource "local_sensitive_file" "bootstrap_script" {
+  filename = "bootstrap.bash"
+  content = templatefile(
+    "${path.module}/bootstrap.bash.tftpl",
+    {
+      NEW_USER             = "user" // TODO(aatifsyed): refactor bootstrap script to not require this
```
Comment (aatifsyed) [inline tf-managed/modules/forest-droplet/main.tf:18]: refactor bootstrap script to not need templating

**Review by elmattic — APPROVED** [review]

**Pull request merged**